test/test_response.py

```diff
@@ -525,6 +525,13 @@ def test_get_case_insensitive_headers(self):
         self.assertEqual(r.headers.get('host'), 'example.com')
         self.assertEqual(r.headers.get('Host'), 'example.com')
 
+    def test_retries(self):
+        fp = BytesIO(b'')
+        resp = HTTPResponse(fp)
+        self.assertEqual(resp.retries, None)
+        resp = HTTPResponse(fp, retries="test")
+        self.assertEqual(resp.retries, "test")
```

Comment:
Put another way: it'd be better if this set `retries` to a value that is more like what it would *actually* be set to.